src/styles/product.scss

```diff
@@ -0,0 +1,93 @@
+:root {
+  margin: 0;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-size: 10px;
+  line-height: 140%;
+}
+
+body,
+p {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+.product {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $card-bgc;
+  border: 1px solid $card-border-color;
+  padding: 32px 16px 16px;
+
+  &__image {
+    display: block;
+    height: 134px;
+    width: 160px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 150%;
+    margin-bottom: 4px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    color: $secondary-text-color;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__rating-reviews {
+    color: $main-text-color;
+    display: flex;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    &-text {
+      color: $secondary-text-color;
+      font-size: 12px;
+      line-height: 150%;
+    }
+
+    &-value {
+      font-weight: 700;
+      font-size: 16px;
+      line-height: 18px;
+      color: $main-text-color;
+    }
+  }
+
+  &__button {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    text-decoration: none;
+    height: 40px;
```

Comment:
it's better not hardcode height for elements with content inside. Just use padding